lib/xnvme_buf.c

```diff
@@ -13,6 +13,7 @@ xnvme_buf_virt_alloc(size_t alignment, size_t nbytes)
 {
 	if (!nbytes) {
 		errno = EINVAL;
+		XNVME_DEBUG("nbytes(%zu)", nbytes);
```

Comment:
I think this is the right idea, however maybe the debug message could be expanded to be more useful.
Maybe something like:
`XNVME_DEBUG("FAILED: invalid value for nbytes: %zu", nbytes);`

Reply:
Agree, fixed!